File: src/rentals/src/main/java/e/library/on/containers/rentals/repository/RentalsEventRepository.java
package e.library.on.containers.rentals.repository;

import e.library.on.containers.rentals.repository.entity.EventEntity;
import org.springframework.data.repository.PagingAndSortingRepository;

import java.util.UUID;

public interface RentalsEventRepository extends PagingAndSortingRepository<EventEntity, UUID> {
}

File: src/rentals/src/main/java/e/library/on/containers/rentals/repository/RentalsEventRepository.kt
package e.library.on.containers.rentals.repository

import e.library.on.containers.rentals.utils.events.Event
import java.time.ZonedDateTime

interface RentalsEventRepository {
    fun addEvent(event: Event)

    fun getAllEventsPast(data: ZonedDateTime?): List<Event>
}
